splunk_connect_for_snmp_poller/manager/poller_utilities.py

```diff
@@ -53,20 +56,31 @@ def onetime_task(inventory_record: InventoryRecord, server_config, splunk_indexe
         inventory_record.to_json(),
         server_config,
         splunk_indexes,
+        None,
         one_time_flag=True,
     )
     return schedule.CancelJob
 
 
-def parse_inventory_file(inventory_file_path):
+def refresh_inventory(inventory_file_path):
+    Path(inventory_file_path).touch()
+
+    return schedule.CancelJob
+
+
+def parse_inventory_file(inventory_file_path, profiles):
     with open(inventory_file_path, newline="") as inventory_file:
         for agent in csv.DictReader(inventory_file, delimiter=","):
             inventory_record = InventoryRecord(
                 agent["host"],
                 agent["version"],
                 agent["community"],
                 agent["profile"],
-                agent["freqinseconds"],
+                profiles["profiles"][agent["profile"]]["frequency"]
+                if profiles
```

Comment:
I would move that to some method with debug message that we dont have frequency and we are using a default one